honeycomb-template/test/benchmark/server/controller/index.suite.js

```diff
@@ -0,0 +1,12 @@
+/* global suite bench */
```

Comment:
test/benchmark/server/*controllers*/index.suite.js instead of test/benchmark/server/*controller*/index.suite.js?